### ISSUE TITLE
[feature] Easy integration with typegraphql

### ISSUE THREAD

**Issue by lookfirst** [Urigo/graphql-scalars#105]
https://typegraphql.ml/docs/scalars.html#custom-scalars

It would be great to somehow automatically populate the scalarsMap. =)

**Comment by ardatan** [comment]
I am not aware of TypeGraphQL. PRs are welcome :)

**Comment by galkin** [comment]
**TL;DR** this issue should be closed

## Details
This package similar to [graphql-iso-date](https://www.npmjs.com/package/graphql-iso-date), [graphql-type-json](https://www.npmjs.com/package/graphql-type-json) is ready for using with `type-graphql`.

There are two cases:
#### 1. Developer explicitly writes type at the type-graphql decorator.
Example:
```typescript
import { EmailAddressResolver, URLResolver } from 'graphql-scalars';
import { Field, ID, ObjectType } from 'type-graphql';

@ObjectType()
export class User {
  @Field(() => ID) id: number;
  @Field() firstName: string;
  @Field() lastName: string;
  @Field(() =>  ) email: string;
  @Field(() => URLResolver) avatarURL: string;
}
```
This code works, so any actions required.
 
#### 2. Developer uses the association map.

Copy-pasted [example](https://typegraphql.ml/docs/scalars.html#custom-scalars):
```typescript
import { ObjectId } from "mongodb";
import { ObjectIdScalar } from "../my-scalars/ObjectId";
import { buildSchema } from "type-graphql";

const schema = await buildSchema({
  resolvers,
  scalarsMap: [{ type: ObjectId, scalar: ObjectIdScalar }],
});
```
`scalarsMap` maps type class to scalar class. You can't use it for map string to URLResolver and EmailAddressResolver. So it's not case for this package.

<hr>
@lookfirst, do I miss something?

**Comment by lookfirst** [comment]
All I was really asking for was a scalarsMap builder, but shrug...
